pegasus/sites.v3/code.org/views/course_table.haml

```diff
@@ -286,3 +264,102 @@
       %br/
       While listed for younger grades, Courses 2–4 can be used by older students
 
+
+
+- tools = []
+- tools << { name: "Artist", starts: 0, ends: 5, description: "Draw cool pictures and designs with the Artist!", xlong: "Use block programming to create images using commands like moveForward and turnLeft.  The resulting programs and images can be shared.", img: "/images/fill-100x100/tutorials/resubmission/artist.jpg" }
```

Comment:
Can remove `xlong` since we're only using one. I don't think we'll need a short/long description for this component.